### ISSUE TITLE
function -w flag doesn't work; must use --wraps

### ISSUE THREAD

**Issue by bitcoinfees** [fish-shell/fish-shell#2772]
According to [the docs](https://fishshell.com/docs/current/commands.html#function) the `-w` flag can be used to specify the wrapped command. However, it doesn't work; only `--w` or `--wraps` does.

### Reproduction Steps:

1. Fresh install of ubuntu trusty and fish 2.2.0
3. Try to use `-w` flag, it doesn't work

### Expected behavior:

```
> function -w git g
    git $argv
end

# no problem
```

### Observed behavior:

```
> function -w git g
    git $argv
end

# the "function" man page is printed 
```

### Additional info:

`--wraps` works properly:

```
> function --wraps git g
    git $argv
end

> g <TAB>  # autocomplete works
```

---
Fish version: 2.2.0

Operating system: Ubuntu 14.04, from PPA

**Comment by ridiculousfish** [comment]
D'oh! Nice find. Fixed as 584cca59bfa5ff038dfecf6471347dde47ff01ed - thanks!

**Comment by faho** [comment]
Wow, I could have sworn this worked. Nice catch!
